docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md

```diff
@@ -11,12 +11,110 @@ menu:
 aliases:
 type: page
 isTocNested: true
-showAsideToc: false
+showAsideToc: true
 ---
 
-You can perform deployment via unidirectional (master-follower) or bidirectional (multi-master) asynchronous replication between data centers.
+Yugabyte Platform allows you to use its UI or API to manage asynchronous replication between independent YugabyteDB clusters. You can perform deployment via unidirectional (master-follower) or bidirectional (multi-master) asynchronous replication between two data centers.
 
-If you are using Yugabyte Platform to manage universes, you need to call the following REST API endpoint on your Yugabyte Platform instance for the producer universe and the consumer universe involved in the asynchronous replication between two data sources:
+Within the concept of replication, universes are divided into the following categories:
+
+- A source universe contains the original data that is subject to replication.
+
+  Note that in the current release, replicating a source universe that has already been populated with data can be done only by contacting Yugabyte Support.
+
+- A target universe is the recepient of the replicated data. One source universe can replicate to one or more target universes.
+
+For additional information on asynchronous replication in YugabyteDB, see [Asynchronous Replication: Overview and Architecture](https://docs.yugabyte.com/latest/architecture/docdb-replication/async-replication/).
+
+## Use the Yugabyte Platform UI
+
+You can use the Yugabyte Platform UI to set up and configure asynchronous replication. In addition, you can perform monitoring by accessing the information about the replication lag and enabling alerts on excessive lag.
+
+### How to set up replication
+
```

Comment:
Worth mentioning you need empty source universe with tables/schema set up, and an empty(?) target universe

Reply:
Rephrased a little